trappy/ftrace.py

```diff
@@ -465,6 +482,38 @@ def __parse_trace_file(self, trace_file):
             raise ValueError('Failed to parse ftrace file {}:\n{}'.format(
                 trace_file, str(e)))
 
+    def __add_warnings(self):
+        # We used to have a bug where when you had a known event whose 'name'
+        # attribute != its 'unique_word', and you specified it explicitly in the
+        # 'events' param, we had two attributes - one for the 'name' and one for
+        # the 'unique_word', and it was undefined which would be populated.
+        # Now we just have the attribute from the 'name'. If there is any code
+        # out there that was relying on this bug (i.e. accessing the
+        # 'unique_word' atribute), this will tell them what they need to do to
+        # fix their code.
+        for name, cls in self.class_definitions.iteritems():
+            # First do some sanity checking (for bugs in TRAPpy)
+            if name != cls.name:
```

Comment:
Can this one actually happen ?

Reply:
No, this is another "should probably be in a test" thing. Will think about it.